remote/remote_state_s3.go

```diff
@@ -443,12 +445,16 @@ func CreateS3BucketWithVersioningSSEncryptionAndAccessLogging(s3Client *s3.S3, c
 	if config.AccessLoggingBucketName != "" {
 		terragruntOptions.Logger.Debugf("Enabling bucket-wide Access Logging on AWS S3 bucket %s - using as TargetBucket %s", config.remoteStateConfigS3.Bucket, config.AccessLoggingBucketName)
 
+		if config.AccessLoggingTargetPrefix != "" {
+			terragruntOptions.Logger.Debugf("Setting Target Prefix for Access Logging to %s", config.AccessLoggingTargetPrefix)
+		}
```

Comment:
NIT: Do we need this log statement here? There's already one in `EnableAccessLoggingForS3BucketWide`.

Reply:
It's definitely not essential - happy to remove.

Reply:
Since we're already logging it, prob no need to duplicate the logic here.